addon/components/qr-scanner.js

```diff
@@ -40,7 +40,7 @@ export default Ember.Component.extend({
       this._scheduleRun(video);
 
     }).catch(error => {
-      this.getWithDefault('onError', Ember.K)(error);
+      this.getWithDefault('onError')(error);
```

Comment:
this would now cause an error if `onError` is not passed into the component. the correct replacement for `Ember.K` would be `() => {}` (an empty function)

Reply:
Added an empty function in place of `Ember.K`. Sorry it took so long; I wasn't able to get around to it till now 😅.